src/simple-length.js

```diff
@@ -38,7 +38,24 @@
     }
   }
 
-  SimpleLength.prototype = shared.LengthValue.prototype;
+  SimpleLength.prototype = Object.create(shared.LengthValue.prototype);
+
+  SimpleLength.prototype.multiply = function(multiplier) {
+    return new SimpleLength((this.value * multiplier), this.type);
+  };
+
+  SimpleLength.prototype.divide = function(divider) {
+    return new SimpleLength((this.value / divider), this.type);
+  };
+
+  SimpleLength.prototype.addSimpleLengths = function(addedLength){
+    return new SimpleLength((this.value + addedLength.value), this.type);
```

Comment:
Please add some validation that the units are the same, or consider moving logic for (eventually) creating CalcLengths into this function.